doc/source/dev/gitwash/development_workflow.rst

```diff
@@ -16,94 +16,81 @@ Basic workflow
 
 In short:
 
-1. Update your ``master`` branch if it's not up to date.
-   Then start a new *feature branch* for each set of edits that you do.
+1. Start a new *feature branch* for each set of edits that you do.
    See :ref:`below <making-a-new-feature-branch>`.
 
-   Avoid putting new commits in your ``master`` branch.
-
 2. Hack away! See :ref:`below <editing-workflow>`
 
-3. Avoid merging other branches into your feature branch while you are
-   working.
-
-   You can optionally rebase if really needed,
-   see :ref:`below <rebasing-on-master>`.
-
-4. When finished:
+3. When finished:
 
    - *Contributors*: push your feature branch to your own Github repo, and
      :ref:`ask for code review or make a pull request <asking-for-merging>`.
```

Comment:
Maybe update this as well. We don't really want people to ask for code review on their Github repo, we only want PRs.